src/components/loader/loader-square.test.tsx

```diff
@@ -99,7 +99,7 @@ test("when inside button, the expected white background colour is applied", () =
 
 // These styling tests required for coverage
 test("when inside button and `isActive` prop is false, the expected background colour is applied", () => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the should be `true`, but as there is a `beforeEach` at the top of the file already you don't need to have any mock value here